code/modules/client/preferences_savefile.dm

```diff
@@ -142,49 +140,46 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 		to_chat(parent, span_danger("[conflicted.category]: [conflicted.full_name] needs updating"))
 
 
-/datum/preferences/proc/load_path(ckey,filename="preferences.sav")
+/datum/preferences/proc/load_path(ckey,filename="preferences.json")
 	if(!ckey)
 		return
-	path = "data/player_saves/[ckey[1]]/[ckey]/[filename]"
+	// path = "data/player_saves/[ckey[1]]/[ckey]/[filename]"
+	path = "data/player_saves_json_testing/[ckey[1]]/[ckey]/[filename]"
+	savefile = new /datum/json_savefile(path)
```

Comment:
I do not agree with this, the proc would have to have its name changed to support this functionality.

Reply:
Also you'll need to audit all usage of this proc, i have a feeling there is a chance for this to be ran when savefile is already initialized.

Reply:
changed it to be a dedicated proc instead